### ISSUE TITLE
Using `require: rubocop/cop/internal_affairs` causes warnings

### ISSUE THREAD

**Issue by splattael** [rubocop/rubocop#13328]
## Expected behavior

The following RuboCop config emits warnings:

```yaml
---
require:
  - rubocop/cop/internal_affairs

AllCops:
  NewCops: enable

InternalAffairs/CopDescription:
  Enabled: false
```

```shell
peter@happy ~/devel/rubocop-test$ rubocop
Warning: InternalAffairs/CopDescription does not support Enabled parameter.

Supported parameters are:

  - Include

Inspecting 0 files


0 files inspected, no offenses detected
```

## Actual behavior

The warning should be no emitted.

## Steps to reproduce the problem

1. Create a directory
2. Copy+paste config above
3. Run `rubocop`

## RuboCop version

```
$ [bundle exec] rubocop -V
peter@happy ~/devel/rubocop-test$ rubocop -V
1.66.1 (using Parser 3.3.5.0, rubocop-ast 1.32.3, running on ruby 3.2.4) [x86_64-linux]
```

I believe this error is due to missing configuration directives in `config/internal_affairs.yml` added by https://github.com/rubocop/rubocop/pull/13212.

We should probably add all possible configuration cop rules and directions similar what we do in `config/default.yml`.
